neurobooth_os/transfer_data.py

```diff
@@ -0,0 +1,32 @@
+"""
+    Moves data from local storage to network storage
+"""
+import subprocess
+import sys
+
+from neurobooth_os import config
+
+server_names = ("presentation", "acquisition", "control")
+
+if len(sys.argv) < 2:
```

Comment:
I would encourage using the argparse standard library instead. See the new iphone dump script or mbient reset script for an example. I don't set a positional required variable in those, but the docs show how to do it.

Reply:
agreed, and done